### PR TITLE
Update EN yml file

### PR DESCRIPTION
**Pull request by GeorgLink**
The update is based on the current metrics website.
All metrics are included in the same order.
New branch and repository names are updated.

Signed-off-by: Georg J.P. Link <linkgeorg@gmail.com>

### CODE REVIEW

**Comment by GeorgLink** [comment]
Unless anything changes during the final weeks of the release, this should be the file to create the PDF.

**Comment by ritik-malik** [comment]
LGTM @GeorgLink!

Should we leave it open for further updation (till the review period)?
Also, I think the default branch for [wg-common](https://github.com/chaoss/wg-common) is `main`

**Comment by GeorgLink** [comment]
> Also, I think the default branch for wg-common is main

good catch. I changed it

**Comment by GeorgLink** [comment]
Should we leave it open for further updation (till the review period)?

> Sure, works for me.

File: automation/active_user_input/english_working-groups-config.yml [hunk 1]
```diff
@@ -16,36 +16,41 @@ end-matter:
...
-  repo-branch: master
+  repo-branch: main
   focus-areas-location: wg-common/focus-areas
   focus-areas:
     contributions:
+    - clones.md
+    - language-distribution.md
     - technical-fork.md
     - types-of-contributions.md
     time:
```
Comment (yash2002109) [inline automation/active_user_input/english_working-groups-config.yml:30]: I don't know if it's intentional to put the focus area `time` before `people` and `place` considering that it doesn't follow the alphabetical order. I see the same kind of issue at a couple of other places. Could you please confirm?

File: automation/active_user_input/english_working-groups-config.yml [hunk 2]
```diff
@@ -16,36 +16,41 @@ end-matter:
...
+  include-wg: True
   wg-fullname: Common Metrics WG
   repo-link: https://github.com/chaoss/wg-common
-  repo-branch: master
+  repo-branch: main
   focus-areas-location: wg-common/focus-areas
   focus-areas:
     contributions:
+    - clones.md
+    - language-distribution.md
```
Comment (yash2002109) [inline automation/active_user_input/english_working-groups-config.yml:27]:
```suggestion
    - programming-language-distribution.md
```

The metric has been recently renamed to match the actual name of the metric.

File: automation/active_user_input/english_working-groups-config.yml [hunk 3]
```diff
@@ -80,10 +85,11 @@ wg-diversity-inclusion:
 #    - channels.md
     - chat-platform-inclusivity.md
     - documentation-accessibility.md
-#    - documentation_discoverability.md
+    - documentation_discoverability.md
```
Comment (yash2002109) [inline automation/active_user_input/english_working-groups-config.yml:88]:
```suggestion
    - documentation-discoverability.md
```
Recently Renamed:
https://github.com/chaoss/wg-dei/blob/main/focus-areas/project-and-community/documentation-discoverability.md

**Review by yash2002109 — COMMENTED** [review]
Hi @GeorgLink, due to some recent changes in the metrics. The YAML file too gets affected. Could you please take a look?

File: automation/active_user_input/english_working-groups-config.yml (hunk 2 above)
Comment (GeorgLink) [reply automation/active_user_input/english_working-groups-config.yml]: Thanks for fixing this!

File: automation/active_user_input/english_working-groups-config.yml (hunk 3 above)
Comment (GeorgLink) [reply automation/active_user_input/english_working-groups-config.yml:88]: Thanks for fixing this!

File: automation/active_user_input/english_working-groups-config.yml (hunk 1 above)
Comment (yash2002109) [reply automation/active_user_input/english_working-groups-config.yml:30]: This makes sense. Thanks for clearing the doubt!

**Comment by yash2002109** [comment]
Thanks for working on this @GeorgLink, I cross-checked with the website. The file looks good to me.

**Comment by ritik-malik** [comment]
Shall we merge it now?

**Comment by ritik-malik** [comment]
Thanks for working on it @GeorgLink !
Merging it

**Pull request merged**